app/models/well.rb

```diff
@@ -16,6 +16,11 @@ class Well < ApplicationRecord
   validates :position, presence: true
   validates :barcode, presence: true, on: :reception
 
+  scope :by_barcode, ->(*barcodes) { joins(:plate).where(plate: { barcode: barcodes }) }
```

Comment:
You can chain scopes but not sure if that is performant.

Reply:
I think they would be the same, but I didnt see a use case where you would want to use individually a by_position() scope to extract all wells for the same position.